src/main/java/bio/terra/service/admin/flight/DrsAliasRegisterStep.java

```diff
@@ -0,0 +1,62 @@
+package bio.terra.service.admin.flight;
+
+import bio.terra.common.iam.AuthenticatedUserRequest;
+import bio.terra.model.DrsAliasModel;
+import bio.terra.service.filedata.DrsDao;
+import bio.terra.service.filedata.DrsDao.DrsAliasSpec;
+import bio.terra.service.filedata.DrsIdService;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import java.util.List;
+import org.apache.commons.collections4.ListUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DrsAliasRegisterStep implements Step {
+  private static final Logger logger = LoggerFactory.getLogger(DrsAliasRegisterStep.class);
+
+  private static final int BATCH_SIZE = 5000;
```

Comment:
Would it make sense to make the batch size configurable?

Reply:
I was debating.  I think given that this is a thing that only we run it's probably better not to pollute the configuration